ansible/deploy.yml

```diff
@@ -140,16 +139,6 @@
           mode: 0700
         tags: [config, peer_keys]
 
-      - name: "Copy node keys to {{ node_config['keys']['dir'] }}"
-        copy:
-          src: "{{ item }}"
-          dest: "{{ project_root }}/{{ node_config['keys']['dir'] }}"
```

Comment:
I need to understand if this is sync/gossip peer keys for seed nodes.

Reply:
I can't really find in the history, but this was the mining keys used to sign the blocks. Probably even before enoise